assignments/lambda-classes.js

```diff
@@ -1 +1,95 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(qualities) {
+        this.name = qualities.name;
+        this.age = qualities.age;
+        this.location = qualities.location;
+        this.gender = qualities.gender;
+    }
+    speak() {
+        return ('Hello my name is'
```

Comment:
Lots of errors on this function. You are using the quotes the wrong way and when `speak` is called it will throw an error. After You are closing the quotation marks JS thins Your function is over and it will ignore rest of the line. There are also some issues with second part of statemant because You are trying to use  ES6 template strings but You are mixing backticks with single quotation.

The proper way would be to use something like this:
```
return `Hello my name is ${this.name} and Im from ${this.location} `;
```